CHANGELOG.md

```diff
@@ -6,8 +6,9 @@
 
 #### Fixes
 
-* [#67](https://github.com/ruby-grape/grape-swagger-entity/pull/67): Various build updates - [@mscrivo](https://github.com/mscrivo).
-* Your contribution here.
+- [#67](https://github.com/ruby-grape/grape-swagger-entity/pull/67): Various build updates - [@mscrivo](https://github.com/mscrivo).
```

Comment:
This causes danger-changelog to break (fix in https://github.com/dblock/danger-changelog/pull/63). These lines need to start with a `*`.

Reply:
I missed that while committing, my editor was auto correcting these to `-`, I went to quickly and thought the rest of the diff in this file (that I didn't commit) was whitespace changes 🤦🏼

Reply:
It exposed a bug in danger-changelog, so I think it's a win!